ext/reflection/php_reflection.c

```diff
@@ -7038,8 +7038,6 @@ static zval *_reflection_write_property(zend_object *object, zend_string *name,
 
 static void reflection_init_class_handlers(zend_class_entry *ce) {
 	ce->create_object = reflection_objects_new;
```

Comment:
Might make sense to inline the function at this point?

Reply:
Yeah either that or remove it, went for inline.

Reply:
Err, by inline I meant remove ^^

Reply:
Ah okay.